tests/node.lisp

```diff
@@ -261,17 +261,29 @@ Finally, all of the nodes constructed by this BLOSSOM-LET are stashed in the pla
                   (address= (gethash left-value dictionary left-value)
                             right-value))
                  (otherwise
-                  (equalp left-value right-value)))))
+                  (equalp left-value right-value))))
+             (blossom-slots (value)
+               (initialize-and-return ((slots nil))
+                 (block quick-exit
+                   (dolist (superclass (closer-mop:class-precedence-list (class-of value)))
+                     (dolist (slot-definition (closer-mop:class-direct-slots superclass))
+                       (push (closer-mop:slot-definition-name slot-definition) slots))
+                     (when (equalp superclass (closer-mop:ensure-class
+                                               'anatevka:blossom-node))
+                       (return-from quick-exit)))))))
```

Comment:
I know I wrote this, but I’m sure this isn’t idiomatic and I can’t remember what the right thing to do is. Some form provides an “implicit tagbody” which one can escape using a bare `(return)`, but…

Reply:
Maybe prog, from https://stackoverflow.com/questions/4394942/which-standard-common-lisp-macros-special-forms-establish-implicit-blocks-named

Reply:
took the `block nil` suggestion